src/instructlab/data/data.py

```diff
@@ -0,0 +1,26 @@
+#from lab import ilab
+import click
+from .generate import generate
+from instructlab import helpers
+
+@click.group(chain=True)
+@click.version_option(package_name="cli")
+@click.option(
+    "--config",
+    "config_file",
+    type=click.Path(),
+    default=helpers.DEFAULT_CONFIG,
+    show_default=True,
+    help="Path to a configuration file.",
+)
```

Comment:
Similar to my comment on the `model` group, we know have `--config` in 3 places instead of just using the top level option. I'd prefer we just keep the top-level one.

Reply:
this was a necessary change not one I really wanted to do. Click has no way of passing the ctx multiple levels, so the ctx with the .obj needs to be initialized on the resource level